.github/workflows/tests.yml

```diff
@@ -105,3 +111,25 @@ jobs:
         uses: ibiqlik/action-yamllint@v3
         with:
           config_file: .yamllint.yml
+
+  upload_coverage:
+    name: "Upload coverage to Codecov"
+    runs-on: "ubuntu-22.04"
```

Comment:
```suggestion
    runs-on: 'ubuntu-latest'
```

Reply:
In the name of consistency should I reformat the quotes of "Remove doctrine/annotations"?